Code/GraphMol/CIPLabeler/Digraph.cpp

```diff
@@ -74,9 +75,12 @@ int Digraph::getNumNodes() const { return d_nodes.size(); }
 
 std::vector<Node *> Digraph::getNodes(Atom *atom) const {
   std::vector<Node *> result;
-  auto queue = std::list<Node *>({getCurrentRoot()});
+  std::deque<Node*> queue = {getCurrentRoot()};
```

Comment:
I completely agree that `std::list` is inappropriate here, but wouldn't it be a bit faster to just use a `std::vector` for queue and do a loop `for(auto idx=0u; idx<queue.size(); ++idx)` ?

Reply:
I tried this with a bunch of ChEMBL molecules and it doesn't make any difference, but maybe you could try with the proteins you were looking at?

Reply:
good call!